docs/reference/glossary.md

```diff
@@ -34,7 +34,11 @@ A command represents an action to be taken or executed. Example commands include
 
 ### Connector
 
-A piece of software that connects the workflow engine with some other system or infrastructure. The Connector might be uni or bidirectional and possibly includes a [job worker](#job-worker). The boundary between Connectors and job workers can be fuzzy, but in general, Connectors connect to other active pieces of software. For example, a DMN Connector might connect to a managed DMN Engine, while a DMN worker will use a DMN library to directly execute decisions.
+A reusable building block that performs the integration with an external system and works out of the box.
+
+The connector might be uni or bidirectional and possibly include a [job worker](#job-worker).
```

Comment:
do we really want to mention job workers here?
we have here an own chapter which describes the differences:
https://docs.camunda.io/docs/components/connectors/introduction-to-connectors/#outbound-connector-vs-job-worker

Reply:
I don't think the added detail is harmful here, particularly as it links to the details for the user if they're new, but interested in additional thoughts here.

Reply:
I've received mixed feedback on this definition in the past, including the DMN component recently - https://camunda.slack.com/archives/C026U8GBNSW/p1682001902222629

I like the inclusion of job worker because it helps communicate the low code strategy by mentioning a topic (job workers) that's been almost exclusively pro-code until connectors.

Besides, we can always polish this up in the future.